doc/build-windows.md

```diff
@@ -38,10 +38,10 @@ To install WSL on Windows 10 with Fall Creators Update installed (version >= 162
   * Enable 'Windows Subsystem for Linux'
   * Click 'OK' and restart if necessary
 2. Install Ubuntu
-  * Open Microsoft Store and search for Ubuntu or use [this link](https://www.microsoft.com/store/productId/9NBLGGH4MSV6)
+  * Open Microsoft Store and search for "Ubuntu 18.04" or use [this link](https://www.microsoft.com/store/productId/9N9TNGVNDL3Q)
   * Click Install
 3. Complete Installation
-  * Open a cmd prompt and type "Ubuntu"
+  * Open a cmd prompt and type "Ubuntu1804"
```

Comment:
Will this stay this way? Or is this temporarily, until `1804` becomes the default

Reply:
I'm not sure, it's a different app in Microsoft Store now.

Reply:
There are multiple different Ubuntu "Apps" available in the Windows store, so I think this should be ok for now.